server/routes/v1/deployment.ts

```diff
@@ -111,3 +113,28 @@ export const postDeployment = [
     })
   },
 ]
+
+export const resetDeploymentApprovals = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const user = req.user
+    const body = req.body as any
+    const deployment: Deployment = await DeploymentModel.findOne({ uuid: body.uuid })
+    if (user?.id !== deployment.metadata.contacts.requester) {
+      throw UnAuthorised({}, 'User is not authorised to do this operation.')
```

Comment:
I'd like to see the deployment that the user is requesting here.  So the first parameter should be `{ deploymentUuid: body.uuid }`.

Reply:
Also, the right hand side of these error messages are user facing, so maybe make it more descriptive (e.g. `You cannot reset the approvals for a deployment you do not own.`).

Reply:
Improved the error text.